### DISCUSSION TITLE
Intermitent issues downloading js files

### DISCUSSION THREAD

**Discussion by ortegaantunes** [cdnjs/cdnjs#14235] [General] [answered]
This could sounds stupid, but there we go. We have a E2E tool, which interact with our application via a CI. Basically, this runs in parallel across a couple of agents. Since one month ago, we started to notice some issues when downloading some required JS files, in this case the page does not even load. This is intermittent, because some tests works as expected and randomly others not. All tests requires those JS files, since this on the main page. My question is, is there any rate limit when download it from cdns.cloudflare.com? From our FW, we don't see any block, but notice some entries for challenge.cloudflare.com.

This is an example https://cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.19.4/jquery.validate.min.js

![Screenshot 2024-03-20 at 15 52 40](https://github.com/cdnjs/cdnjs/assets/29414973/59aafe34-9321-4a79-befa-faa73dc12d0a)

**Comment by MattIPv4** [comment]
👋 Would you be able to provide the response headers for the bad responses you're seeing?

**Reply by ortegaantunes** [reply]
Thanks @MattIPv4. The response header is empty. 

![Screenshot 2024-03-20 at 16 23 14](https://github.com/cdnjs/cdnjs/assets/29414973/970d6406-abad-4cf9-8b29-f0aa3201fe9a)

**Reply by MattIPv4** [reply]
🤔 I don't think your test suite is actually making the request then? What status code are you seeing, what error are you seeing logged for the network request?

**Reply by ortegaantunes** [reply]
Yeah, I think so. There is no error for the network request, so looks like the request is not being "triggered"

**Reply by MattIPv4** [reply]
👍 I'll leave that for you to investigate then, sounds like it isn't an issue with cdnjs unless I'm missing something.

**Reply by ortegaantunes** [reply, marked as answer]
@MattIPv4 we can close this issue. This is an issue on the E2E framework side. Thanks.
